qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/presenter.py

```diff
@@ -18,7 +17,7 @@
 from .settings.settings_model import SettingsModel
 from .settings.settings_view import SettingsView
 from .settings.settings_presenter import SettingsPresenter
-from .settings.settings_helper import get_setting, set_setting
+from mantidqtinterfaces.Engineering.gui.engineering_diffraction.settings.settings_helper import get_setting, set_setting
```

Comment:
This is nit picking really - but does this line need changing? The other imports seem to work with a relative path

Reply:
Yep - I think that import line was added automatically by pycharm - I'll make it relative